bigbluebutton-html5/private/config/settings.yml

```diff
@@ -472,13 +473,19 @@ public:
     typingIndicator:
       enabled: true
     moderatorChatEmphasized: true
-  note:
+  notes:
```

Comment:
Just drawing attention as this will trip some people. No change needed in the code, just commenting for reference